reactivesearch/src/ubi/UbiEvent.js

```diff
@@ -0,0 +1,41 @@
+class UbiEvent {
+  /**
+   * This maps to the UBI Specification at https://github.com/o19s/ubi
+   */
+  constructor(action_name, client_id, session_id, query_id, event_attributes, message = null) {
+    this.application = "Chorus"
+    this.action_name = action_name;
+    this.query_id = query_id;
+    this.session_id = session_id;        
+    this.client_id = client_id;
+    this.user_id = '';
+    this.timestamp = new Date().toISOString();
```

Comment:
I can't remember. Does this add the trailing `Z`?

Reply:
Yes, we checked that. What we had before did not do what we wanted. This line has the `T` in the middle and the trailing `Z`